### ISSUE TITLE
tflint azurerm reports false positive with AEH premium SKU

### ISSUE THREAD

**Issue by zefferno** [terraform-linters/tflint-ruleset-azurerm#181]
Hello, 

I'd like to report a bug with the azurerm provider regarding Azure EventHub running premium SKU.  Premium SKU is a supported SKU type with Azure Event Hub.

tflint & tflint azurerm ruleset versions:
```
$ tflint --version
TFLint version 0.38.1
+ ruleset.azurerm (0.17.0)
```

tflint reports an error:
```
$ tflint -f compact --module --no-color
1 issue(s) found:

main.tf:19:12: Error - "Premium" is an invalid value as sku (azurerm_eventhub_namespace_invalid_sku)
```

Example of Terraform code to reproduce:
```
resource "azurerm_eventhub_namespace" "test123" {
  ...
  sku                  = "Premium"
  ...
}
```

Relevant official docs:
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/eventhub_namespace#argument-reference
- https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas

**Comment by wata727** [comment]
Fixed by https://github.com/terraform-linters/tflint-ruleset-azurerm/pull/184
